api/envoy/protobuf/descriptor.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package envoy.protobuf;
```

Comment:
drive by API nit: Is it worth having some type of "PII" namespace that we would use for this? I think we are going to wind up with other annotations in this area. Ultimately I also think this should go into UDPA but will defer to @htuch on whether to do that now or later.

Reply:
I think we should do it now. We already have an annotations tree in UDPA, see https://github.com/cncf/udpa/tree/master/udpa/annotations

Reply:
Sounds reasonable. I'll try to find you tomorrow for help with that.

Reply:
https://github.com/cncf/udpa/pull/17